rust/processor/src/processors/parquet_processors/parquet_user_transactions_processor.rs

```diff
@@ -78,6 +78,52 @@ impl Debug for ParquetUserTransactionsProcessor {
         )
     }
 }
+pub async fn process_transactions(
```

Comment:
nit: new line

Reply:
weird that the formatter didn't get this

Reply:
yeah formatter sometimes misses, `cargo +nightly fmt  && cargo +nightly xclippy` I run this just in case.